frontend/src/ui-components/AcmTable/AcmTable.tsx

```diff
@@ -1543,47 +1622,128 @@ export function AcmTable<T>(props: AcmTableProps<T>) {
+                          ? mergeProps(...transforms.map((transform: ITransform) => transform()))
+                          : []
+                        const sortProps = column.sort
+                          ? {
+                              sort: {
+                                sortBy: adjustedSort || {},
+                                onSort: (_event: React.MouseEvent, index: number, direction: SortByDirection) => {
+                                  return updateSort({ index, direction })
+                                },
+                                columnIndex: columnIndex,
+                              },
+                            }
+                          : {}
+                        const tooltipProps = column.tooltip
+                          ? {
+                              tooltip: column.tooltip,
+                              tooltipProps: { position: TooltipPosition.left },
+                              info: {
+                                popover: column.tooltip,
+                              },
+                            }
+                          : undefined
+                        return (
+                          <Th
+                            key={column.id}
+                            dataLabel={column.header}
+                            {...tooltipProps}
+                            {...sortProps}
+                            {...iTransformColumnProps}
+                          >
+                            {column.header}
+                          </Th>
+                        )
+                      })}
+                  </Tr>
+                </Thead>
+                {primaryRows.map((row, rowIndex) => {
+                  return (
+                    <Tbody isExpanded={row.isOpen} key={`${row.props.key}-tablebody`}>
+                      <Tr key={`${row.props.key}-tablerow`} ouiaId={row?.props?.key}>
+                        {onCollapse && (
+                          <Td expand={{ isExpanded: row.isOpen || false, rowIndex, onToggle: onCollapse }}></Td>
+                        )}
+                        {hasSelectionColumn && (
+                          <Td
+                            key={`${row.props.key}-select`}
+                            select={{
+                              rowIndex,
+                              onSelect: onSelectCallback(),
+                              isSelected: selected[row.props.key],
+                              isDisabled: row.disableSelection,
+                            }}
+                          />
+                        )}
+                        {row?.cells?.map((cell, cellIndex) => {
+                          // setup row props, including celltransforms
+                          const transforms: ITransform[] | undefined = selectedSortedCols[cellIndex]?.cellTransforms
+                          const iTransformCellProps = transforms
+                            ? mergeProps(
```

Comment:
I was surprised that the transforms do not seem to be deprecated, but I did not find examples from PF of using them. I wonder if we should open a follow up task to look at all the transforms we are using and seeing if we can move to the new way of doing them?

Reply:
[I looked into that as well](https://patternfly.slack.com/archives/C4FM977N0/p1744808941706229?thread_ts=1744783266.728319&cid=C4FM977N0). I'm not so sure there is a "new way" as much as there is no specified way of doing this from PatternFly any longer. Even [the example given from this PF engineer](https://patternfly.slack.com/archives/C4FM977N0/p1744808941706229?thread_ts=1744783266.728319&cid=C4FM977N0) does not actually use an itransform type.

To your other point though, I agree that it might be worth iterating on a unified pattern for ACM console's transformation needs. Opened a task here: https://issues.redhat.com/browse/ACM-20735